cosmos/runtime/app.go

```diff
@@ -305,6 +305,8 @@ func NewPolarisApp( //nolint: funlen // from sdk.
 	// THE "DEPINJECT IS CAUSING PROBLEMS" SECTION
 	// ===============================================================
 
+	// Pntr to a Pntr
+	dk := &app.DistrKeeper
```

Comment:
don't need the double pointer here because the keeper's are already initialized by this point